website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -0,0 +1,157 @@
+---
+layout: docs
+page_title: Audit Filtering
+description: |-
+  How to filter audit entries in Vault.
+---
+
+[filter syntax]: /vault/docs/concepts/filtering
+
+# Filtering audit entries
+
+This page describes how to use filters with Vault audit devices to fine tune which
+audit entries should be written to an audit log.
+
+Starting in Vault 1.16.0, audit devices can be enabled with a `filter` option, which
+is used to evaluate audit entries and decide if they should be written to a particular
+audit log. Please ensure you are familiar with the [filtering concept](/vault/docs/concepts/filtering) before
+attempting to configure an audit device with a filter.
+
+<Note title="Advanced audit feature">
+  The use of filtering in Vault's audit system should be considered an advanced feature.
+  Whilst it has been designed to be simple and flexible, exclusively enabling filtered
+  devices could result in some requests and responses that are not audited.
+</Note>
+
+Please see the [Vault security model](/vault/docs/internals/security) for further information.
```

Comment:
I'm a little confused by this sentence -- we don't modify this file and it doesn't mention auditing in detail. This sentence feels like a non sequitur and I think we can probably remove it?

Reply:
I wanted to include it in the `<note>` but it doesn't support the formatting. 😞 

The main reason being that the security model explains that all requests/responses to Vault will be audited (when audit is on), but through filtering it is possible for that not to happen, so I figured drawing attention to the page here might make sense 🤷🏼 🤔

Reply:
Hmm, yeah that makes sense. I'm definitely not opposed to leaving this in, but it does read a little strange right now. How about...

`To understand how auditing fits into the [Vault security model](/vault/docs/internals/security), please consult that documentation for further information.`

Though, it's fine as-is, just a suggestion :)

Reply:
I've updated it somewhere in between, thanks for the suggestions - they were really useful.